### PR TITLE
API-11989 Add EncounterType to DatamartEncounter and tests and samples

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

**Review by jhulbert-va — APPROVED** [review]

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/encounter/EncounterSamples.java [hunk 1]
```diff
@@ -40,6 +41,23 @@ public DatamartEncounter encounter(String cdwId, String patientIcn) {
...
+              EncounterType.builder()
+                  .coding(
+                      Optional.of(
+                          DatamartCoding.builder()
+                              .code(Optional.of("99212"))
+                              .display(Optional.of("OFFICE O/P EST SF 10-19 MIN"))
+                              .build()))
+                  .text(
+                      Optional.of(
+                          "OFFICE OR OTHER OUTPATIENT VISIT FOR THE EVALUATION AND MANAGEMENT OF AN "
```
Comment (jblefkove-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/encounter/EncounterSamples.java:54]: For the purposes of test data, let's shorten this to something more reasonable

**Comment by jtaylor-va** [comment]
Is there an test record for Inpatient Encounters?

**Comment by jblefkove-va** [comment]
> Is there an test record for Inpatient Encounters?

For unit testing I think we only need one; we can make sure Inpatient and Outpatient Encounters are both represented in the sandbox data.

**Review by jblefkove-va — APPROVED** [review]
